types/gh-pages/index.d.ts

```diff
@@ -3,8 +3,27 @@
 // Definitions by: Daniel Rosenwasser <https://github.com/DanielRosenwasser>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
+export interface Git {
+    cwd: string;
+    cmd: string;
+    output: string;
```

Comment:
aren't those internal? (undocumented) Those seems to be set on object creation only via function params.

Reply:
Good eye – removed